scripts/sv/callVariants.sh

```diff
@@ -21,12 +21,11 @@ echo "Assuming 2-bit reference: " "$TWOBIT_REFERENCE_LOCATION"
 
 cd "$GATK_DIR" 
 
-./gatk-launch CallVariantsFromAlignedContigsSpark \
-    --inputAlignments "$PROJECT_OUTPUT_DIR"/aligned_assemblies \
-    --inputAssemblies "$PROJECT_OUTPUT_DIR"/assembly_0 \
+./gatk-launch CallVariantsFromAlignedContigsSAMSpark \
```

Comment:
The SAM version was just there so that I could harness the call variants step using a BAM file as input as opposed to our internal AlignmentRegion representation. If we're moving to a SAM representation for everything, we should probably just be able to have one tool.